templates/base/head_script.tmpl

```diff
@@ -47,3 +47,12 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 	{{/* in case some pages don't render the pageData, we make sure it is an object to prevent null access */}}
 	window.config.pageData = window.config.pageData || {};
 </script>
+
+<script type="module">
+	const browserSyncJson = JSON.stringify({lc: window.navigator.language, tz: new Date().getTimezoneOffset()});
```

Comment:
We don't need to communicate `window.navigator.language`, as we already have the language from the chosen language setting (reflected in `html[lang]`).

Reply:
That's another question needs to be discussed. `window.navigator.language` is user browser's default language, which is different from the user chosen language for Gitea `document.documentElement.lang`. 

So I just report the navigator language as it is, then backend can know: browser's default language, user chosen language.

Reply:
I think the only place we should rely on `window.navigator.language` is to determine initial language for users that have set none yet, e.g. a `auto` language could be added that defaults to `window.navigator.language`.